projects/components/src/header/application/application-header.component.scss

```diff
@@ -45,12 +45,17 @@
     padding-left: 46px;
   }
 
-  .right-side {
+  .right-side-content {
     display: flex;
     margin-left: auto;
     align-items: center;
   }
 
+  .left-side-content {
+    display: flex;
+    align-items: center;
+  }
```

Comment:
should we use justify content: flex-start here?

Reply:
that's the default, it's already behaving that way.